demos/projects/ST/stm32h745i-disco/cm7/port/azure_iot_flash_platform.c

```diff
@@ -0,0 +1,229 @@
+/* Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License. */
+
+#include <string.h>
+#include "azure_iot_flash_platform.h"
+#include "azure_iot_flash_platform_port.h"
+#include "stm32h7xx_hal.h"
+/* Logging */
+#include "azure_iot.h"
+#include "azure/core/az_base64.h"
+#include "mbedtls/md.h"
+
+#define azureiotflashH745_WORD_SIZE    32
```

Comment:
```suggestion
#define azureiotflashH745_WORD_SIZE    sizeof( long )
```
would this work to be a bit less magic-number-y? (not sure if this would be 2* or just one long)

Reply:
32 works but `sizeof(long)` doesn't. I think "word" in this case is 32 bytes.